src/components/views/rooms/MemberInfo.js

```diff
@@ -481,22 +489,22 @@ module.exports = WithMatrixClient(React.createClass({
     },
 
     _calculateCanPermissions: function(me, them, powerLevels) {
```

Comment:
Agreed this should definitely be in the js-sdk. Trying to put some limit on the depth of this rabbit hole traversal though.